script/publish_views

```diff
@@ -14,6 +15,8 @@ from google.api_core.exceptions import BadRequest
 from google.cloud import bigquery
 import sqlparse
 
+sql_root = Path(__file__).resolve().parent.parent / "sql"
+assert sql_root.exists(), f"{sql_root} not found"
```

Comment:
Having an assert when importing makes me slightly nervous, but I suppose it's true that this invariant should hold and that it's better to raise this message than have the logic fail more mysteriously.

Reply:
This was also necessary for testing, since there isn't a great way to prod into the behavior other than running the script itself and inserting print statements. I rather it fail here than mysteriously during automation.